code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -2,8 +2,8 @@
 //////////////////////////Poison stuff (Toxins & Acids)///////////////////////
 
 /datum/reagent/toxin
-	name = "Toxin"
-	description = "A toxic chemical."
+	name = "Space Toxin"
+	description = "A spooky poisonous chemical."
```

Comment:
Oh noes, a spooky chemical!

Reply:
I don't like that it's just named Toxin, it's kinda lame.